api/routes/issues.js

```diff
@@ -0,0 +1,185 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+
+const Issue = require('../models/issue');
+const Comment = require('../models/comment');
+
+// GET / - list all created issues
+router.get('/', (req, res, next) => {
+    Issue.find()
+        .select('-__v')
+        .exec()
+        .then(docs => {
+            const response = {
+                count: docs.length,
+                issues: docs.map(doc => {
```

Comment:
Here you are doing some transformation of the issue before you return it. What is the reason for introducing the `request` attribute in the response? Why not just return `doc` as is?